### ISSUE TITLE
Warning: This synthetic event is reused for performance reasons. If you're seeing this, you're adding a new property in the synthetic event object. The property is never released. See https://fb.me/react-event-pooling for more information.

### ISSUE THREAD

**Issue by artursgirons** [zillow/react-slider#76]
With React v15.0.1

**Comment by ghost** [comment]
I am also seeing this warning the first time I move the slider.

**Comment by klase** [comment]
+1

**Comment by miraage** [comment]
@mpowaga any progress?

**Comment by AndersDJohnson** [comment]
Also seeing this. Any solution yet? What about #73?

`npm ls`:

```
├── autoprefixer@6.3.6
├── babar@0.0.3 extraneous
├── babel-cli@6.10.1
├── babel-core@6.9.1
├── babel-loader@6.2.4
├── babel-preset-es2015@6.9.0
├── babel-preset-react@6.5.0
├── babelify@7.3.0
├── bootstrap@3.3.6
├── bootstrap-slider@7.1.0 extraneous
├── brfs@1.4.3 extraneous
├── browserify@13.0.1
├── chart.js@2.1.6
├── css-loader@0.23.1
├── file-loader@0.9.0
├── geojson-merge@0.1.0
├── jquery@3.0.0
├── json-loader@0.5.4
├── jsonfile@2.3.1
├── leaflet@0.7.7
├── lodash@4.13.1
├── numeral@1.5.3 extraneous
├── pad@1.0.0 extraneous
├── postcss-loader@0.9.1
├── precss@1.4.0
├── react@15.1.0
├── react-dom@15.1.0
├── react-slider@0.6.1
├── strftime@0.9.2
├── style-loader@0.13.1
├── url-loader@0.5.7
├── watch@0.18.0
├── webpack@1.13.1
├── webpack-dev-server@1.14.1
└── webpack-livereload-plugin@0.8.1
```

**Comment by AndersDJohnson** [comment]
#73

**Comment by jamesbrauman** [comment]
This has been addressed in this commit https://github.com/mpowaga/react-slider/commit/6fee17df90fa8303335695c87ca8908937afc8e9.

I will publish a new version to npmjs.com when I have received appropriate permissions from @mpowaga to do so.

**Comment by jamesbrauman** [comment]
@artursgirons This issue has been resolved in the latest release `0.7.0`.
